converter/shieldhit/parser.py

```diff
@@ -51,6 +57,16 @@ def get_configs_json(self) -> dict:
 class ShieldhitParser(DummmyParser):
     """A regular SHIELD-HIT12A parser"""
 
+    def __init__(self) -> None:
+        super().__init__()
+        #Add version variable to deploy script
```

Comment:
```suggestion
        # Add version variable to deploy script
```
This should make deepsource happier